experimental/generation/generator/packages/library/templates/standard/form-BeginDialog.dialog.lg

```diff
@@ -13,18 +13,61 @@
             "$kind": "Microsoft.SendActivity",
             "activity": "\${welcome()}"           
         },
-        {
-            "$kind": "Microsoft.IfCondition",
-            "condition": "!turn.activityProcessed",
-            "actions": [
-                {
-                    "$kind": "Microsoft.EmitEvent",
-                    "eventName": "activityReceived",
-                    "eventValue": "=turn.activity",
-                    "bubbleEvent": false
-                }
-            ]
-        }
+        ${setActions()}
     ]
 }
 ```
+
+# setActions
+- IF: ${!empty(where(schema.properties, property, property.value.$dynamiclist == "true"))}
```

Comment:
>IF: ${!empty(where(schema.properties, property, property.value.$dynamiclist == "true"))} [](start = 2, length = 88)

I would move this inside of setPropertiesForDynamicLists where if it is true you do ", <value>" otherwise an empty string.  That way you don't have to duplicate the body twice.